File: src/main/java/backend/controller/StaticController.java
package backend.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller
public class StaticController {
    @RequestMapping(value = "/", method = RequestMethod.GET)
    public String apiDoc() {
        return "redirect:swagger-ui.html";
    }
}

File: src/main/java/backend/controller/StaticController.kt
package backend.controller

import backend.util.Profiles
import org.springframework.context.annotation.Profile
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import springfox.documentation.annotations.ApiIgnore

@Profile(Profiles.DEVELOPMENT)
@Controller
class StaticController {

    @ApiIgnore
    @GetMapping("/")
    fun apiDoc(): String {
        return "redirect:swagger-ui.html"
    }
}
